tests/sources/tools/control/mobile_manipulation/test_mobile_manipulation.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import os
+import shutil
+import torch
+import unittest
+from opendr.control.mobile_manipulation.mobileRL.utils import create_env
+from opendr.control.mobile_manipulation.mobile_manipulation_learner import MobileRLLearner
+from pathlib import Path
+
+TEST_ITERS = 3
+TEMP_SAVE_DIR = Path(__file__).parent / "mobile_manipulation_tmp"
+EVAL_ENV_CONFIG = {
+    'env': 'pr2',
+    'penalty_scaling': 0.01,
+    'time_step': 0.02,
+    'seed': 42,
+    'strategy': 'dirvel',
+    'world_type': 'sim',
+    'init_controllers': False,
+    'perform_collision_check': True,
+    'vis_env': False,
+    'transition_noise_base': 0.0,
+    'ik_fail_thresh': 20,
+    'ik_fail_thresh_eval': 100,
+    'learn_vel_norm': -1,
+    'slow_down_real_exec': 2,
+    'head_start': 0,
+    'node_handle': 'train_env'
+}
+
+
+def get_first_weight(learner):
+    return list(learner.stable_bl_agent.get_parameters()['policy'].values())[0].clone()
+
+
+class MobileManipulationTest(unittest.TestCase):
+    learner = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.env = create_env(EVAL_ENV_CONFIG, task='rndstartrndgoal', node_handle="train_env", wrap_in_dummy_vec=True,
```

Comment:
It would be good to print the test name just at the beginning, something like
```suggestion
        print("\n\n**********************************\nTEST Mobile RL Learner\n"
              "**********************************")
        cls.env = create_env(EVAL_ENV_CONFIG, task='rndstartrndgoal', node_handle="train_env", wrap_in_dummy_vec=True,
```